src/js/actions/appActions.js

```diff
@@ -178,8 +178,12 @@ export const setVersionInfo = info => (dispatch, getState) =>
     })
   );
 
+const versionRegex = new RegExp(/\d+\.\d+/);
```

Comment:
Q: Why not define it in the scope where it's used only?

Reply:
to squeeze out the precious performance improvement of only initializing this once on load and not on call to the function... 😅

Reply:
:laughing: :light_rail: